src/renderer/scss-partials/_ft-list-item.scss

```diff
@@ -283,6 +283,7 @@ $watched-transition-duration: 0.5s;
     flex-direction: column;
     min-height: 230px;
     padding-block-end: 20px;
+    height: 91.46%;
```

Comment:
This one is better
```suggestion
    // Equal heights in grid
    // 100% height with `border-box` (otherwise too high)
    box-sizing: border-box;
    height: 100%;
```

Reply:
I agree!